url_test.go

```diff
@@ -83,7 +83,51 @@ func TestRedactURL(t *testing.T) {
 				Path:     "hashicorp/go-getter-test-private.git",
 				RawQuery: "sshkey=",
 			},
-			want: "ssh://git@github.com/hashicorp/go-getter-test-private.git?sshkey=",
+			want: "ssh://git@github.com/hashicorp/go-getter-test-private.git?sshkey=redacted",
+		},
+		{
+			name: "multiple SSH keys in URL query parameter",
+			url: &url.URL{
+				Scheme:   "ssh",
+				User:     url.User("git"),
+				Host:     "github.com",
+				Path:     "repo.git",
+				RawQuery: "sshkey&sshkey=secretkey",
```

Comment:
Suggestion: Can this be `sshkey=secretkey1&sshkey=secretkey2`? Right now, it's the same as the test `multiple SSH keys with empty and non-empty values`.

Reply:
Adjusted the tests in [80a526c](https://github.com/hashicorp/go-getter/pull/528/commits/80a526cd1b22f0b770dfd158557341f890f3b4a7) also added one with multiple ssh key values which was missing